plugins/woocommerce/includes/rest-api/Controllers/Version1/class-wc-rest-product-reviews-v1-controller.php

```diff
@@ -144,6 +144,11 @@ public function get_items_permissions_check( $request ) {
 	 * @return WP_Error|boolean
 	 */
 	public function get_item_permissions_check( $request ) {
+		$review = $this->get_review( (int) $request['id'], (int) $request['product_id'] );
```

Comment:
do you think we also have a check for returning 404 when the review is null? I don't see it causing any issues _now_ since there's a blanket `moderate_comments` or `edit_products` capability check.